std/experimental/allocator/common.d

```diff
@@ -92,13 +92,14 @@ size_t goodAllocSize(A)(auto ref A a, size_t n)
 Returns s rounded up to a multiple of base.
 */
 @safe @nogc nothrow pure
-package size_t roundUpToMultipleOf(size_t s, uint base)
+size_t roundUpToMultipleOf(size_t s, uint base)
```

Comment:
This exposes `roundUpToMultipleOf` to the docs, it's used in many of the building blocks and the documented examples in `quantizer` needs it. An alternative would be to use a more simplistic `Quantizer` than:

```d
alias MyAlloc = Quantizer!(
     FreeTree!GCAllocator,
      n => n.roundUpToMultipleOf(n <= 16_384 ? 64 : 4096));
``